Dwarf.cpp

```diff
@@ -1,7 +1,23 @@
 #include "Dwarf.h"
+#include "Utility.h"
 
-//Dwarf::Dwarf
+Dwarf::Dwarf(std::string charName, int hp, int armor) : Character(hp, armor, 4), name(charName) 
+{
+    helpfulItems = makeHelpfulItems(3);
+    defensiveItems = makeDefensiveItems(2);
+}
 
-//Dwarf::getName
+const std::string& Dwarf::getName()
+{
+    return name;
+}
 
-//Dwarf::getStats
\ No newline at end of file
+std::string Dwarf::getStats()
+{
+    return getCharacterStats(this);
+}
+
+void Dwarf::attack(Character& other)
+{
+    Character::attack(other);
+}
```

Comment:
Don't override base-class member functions unless you are going to change the behavior that happens when that function is called.
- [ ] remove this function